framework/helpers/BaseInflector.php

```diff
@@ -231,13 +231,53 @@ class BaseInflector
         'ÿ' => 'y',
     ];
+
     /**
-     * @var mixed Either a [[Transliterator]] or a string from which a [[Transliterator]]
-     * can be built for transliteration used by [[slug()]] when intl is available.
-     * @see http://php.net/manual/en/transliterator.transliterate.php
+     * Shortcut for `Any-Latin; NFKD` transliteration rule. The rule is strict, letters will be transliterated with
+     * the closest sound-representation chars. The result my contain any UTF-8 chars. For example:
+     * `获取到 どちら Українська: ґ,є, Српска: ђ, њ, џ! ¿Español?` will be transliterated to
+     * `huò qǔ dào dochira Ukraí̈nsʹka: g̀,ê, Srpska: đ, n̂, d̂! ¿Español?`
+     *
+     * Used in [[transliterate()]].
+     * For detailed information see [unicode normalization forms](http://unicode.org/reports/tr15/#Normalization_Forms_Table)
+     * @see http://unicode.org/reports/tr15/#Normalization_Forms_Table
+     * @see transliterate()
      */
-    public static $transliterator = 'Any-Latin; NFKD';
+    const TRANSLITERATE_STRICT = 'Any-Latin; NFKD';
 
+    /**
+     * Shortcut for `Any-Latin; Latin-ASCII` transliteration rule. The rule is loose, letters will be
+     * transliterated to characters of Latin-1 ASCII table. For example:
+     * `获取到 どちら Українська: ґ,є, Српска: ђ, њ, џ! ¿Español?` will be transliterated to
+     * `huo qu dao dochira Ukrainsʹka: g,e, Srpska: d, n, d! ¿Espanol?`
+     *
+     * Used in [[transliterate()]].
+     * For detailed information see [unicode normalization forms](http://unicode.org/reports/tr15/#Normalization_Forms_Table)
+     * @see http://unicode.org/reports/tr15/#Normalization_Forms_Table
+     * @see transliterate()
+     */
+    const TRANSLITERATE_MEDIUM = 'Any-Latin; Latin-ASCII';
+
+    /**
+     * Shortcut for `Any-Latin; Latin-ASCII; [\u0080-\uffff] remove` transliteration rule. The rule is strict, letters will be transliterated with
+     * the closest sound-representation chars. The result my contain any UTF-8 chars. For example:
+     * `获取到 どちら Українська: ґ,є, Српска: ђ, њ, џ! ¿Español?` will be transliterated to
+     * `huo qu dao dochira Ukrainska: g,e, Srpska: d, n, d! Espanol?`
+     *
+     * Used in [[transliterate()]].
+     * For detailed information see [unicode normalization forms](http://unicode.org/reports/tr15/#Normalization_Forms_Table)
+     * @see http://unicode.org/reports/tr15/#Normalization_Forms_Table
+     * @see transliterate()
+     */
+    const TRANSLITERATE_LOOSE = 'Any-Latin; Latin-ASCII; [\u0080-\uffff] remove';
+
+
+    /**
+     * @var mixed Either a [[\Transliterator]], or a string from which a [[\Transliterator]] can be built
+     * for transliteration. Used by [[transliterate()]] when intl is available. Defaults to [[TRANSLITERATE_LOOSE]]
+     * @see http://php.net/manual/en/transliterator.transliterate.php
+     */
+    public static $transliterator = self::TRANSLITERATE_LOOSE;
```

Comment:
Nobody can guarantees that transliteration or formatting using i18n will be steady release-to-release because each ICU update changes the rules of formatting or transliteration for languages and nobody complains on it.
My change does not break the transliteration, as you see. It just gives a little more precise result for some cases.